orion-server/src/main/java/com/pinterest/orion/core/automation/operator/kafka/BrokerHealingOperator.java

```diff
@@ -160,6 +161,13 @@ public void operate(KafkaCluster cluster) throws Exception {
           "Orion agents on " + cluster.getClusterId() + " are unhealthy, no URPs on the cluster: " + alertableUnhealthyAgentBrokersWithoutURPs,
           "orion"
       ));
+      OrionServer.metricsCounterInc(
+              "BrokerHealing.OrionAgent",
```

Comment:
Can we use the same format for these? They ideally should have the same number of `.`'s in them.

Reply:
OK. Will update